src/sql/workbench/contrib/notebook/browser/cellViews/placeholderCell.component.html

```diff
@@ -7,7 +7,7 @@
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div class="placeholder-cell-component" style="flex: 0 0 auto;">
 		<div class="placeholder-cell-component text">
-			<p>{{clickOn}} <a id="addCode" href="#" (click)="addCell('code', $event)">{{plusCode}}</a> {{or}} <a id="addMarkdown" href="#" (click)="addCell('markdown', $event)">{{plusText}}</a> {{toAddCell}}</p>
+			<p>{{clickOn}} <a id="addCode" href="#" role="button" (click)="addCell('code', $event)">{{plusCode}}</a> {{or}} <a id="addMarkdown" href="#" role="button" (click)="addCell('markdown', $event)">{{plusText}}</a> {{toAddCell}}</p>
```

Comment:
please also add aria-label

Reply:
the aria-label should be descriptive like 'add a code cell' and 'add a text cell'